aws-auth-cognito/src/test/java/com/amplifyframework/auth/cognito/AWSCognitoAuthPluginTest.kt

```diff
@@ -681,6 +683,53 @@ class AWSCognitoAuthPluginTest {
         verify(timeout = CHANNEL_TIMEOUT) { realPlugin.clearFederationToIdentityPool(any(), any()) }
     }
 
+    @Test
+    fun setUpTOTP() {
```

Comment:
For this test and the 3 tests below it, `any()` shouldn't be used on the last blocks. Its should be asserting that the correct listeners, etc are passed. In this case it should check expectedOnSuccess and expectedOnError.